src/main/java/com/faforever/client/patch/GameUpdaterImpl.java

```diff
@@ -99,6 +95,19 @@ public GameUpdater addFeaturedModUpdater(FeaturedModUpdater featuredModUpdater)
             createFaPathLuaFile(initFile.getParent().getParent());
             copyLegacyInitFile(initFile);
           }
+        })
+        .exceptionally(throwable -> {
+          boolean allowReplaysWhileInGame = preferencesService.getPreferences().getForgedAlliance().isAllowReplaysWhileInGame();
+          if (throwable.getCause().getCause() instanceof AccessDeniedException && allowReplaysWhileInGame) {
+            log.info("Unable to update file cause they are not writeable. " +
+                "Experimental feature is turned on " +
+                "that allows multiple game instances to run in parallel this is most likely the cause.");
+            throw new UnsupportedOperationException("Unable to patch Forged Alliance to the required version due to conflicting version running");
+          } else if (!(allowReplaysWhileInGame && version != null)) {
+            log.warn("Game not terminated correctly", throwable);
+            notificationService.addImmediateErrorNotification(throwable, "error.game.notTerminatedCorrectly");
```

Comment:
This message is not really up to date anymore. it could really be anything that caused it but the game not terminating correctly should not be an issue anymore